### ISSUE TITLE
Outdated dependencies in package.json

### ISSUE THREAD

**Issue by seifferth** [brrd/Abricotine#270]
There seem to be quite a few old versions specified as dependencies in `package.json`. Is there a particular reason, why you do not use the `"^x.x.x"` notation for specifying any version greater than a given version?

Building with the latest versions of all dependencies on Arch Linux seems to work fine for me. I would be happy to adjust the versions in `package.json` to use `"^x.x.x"` and create a pull request, if there are no objections to using latest versions by default.

Edit: Typo

**Comment by brrd** [comment]
> There seem to be quite a few old versions specified as dependencies in package.json. Is there a particular reason, why you do not use the "^x.x.x" notation for specifying any version greater than a given version?

A couple of years ago CodeMirror accidentally introduced some bug in a minor release. It looked very weird because we observed different behaviors with the same version of Abricotine, depending on when the app was built by the user. After a while we finally understood that it came from the "^x.x.x" notation. This is why I decided to remove it. Now we're sure Abricotine uses the same dependencies everywhere, which makes debugging much easier IMHO.

So it's is nice to offer but I prefer to keep it this way.
